testing/tiptop/APKBUILD

```diff
@@ -0,0 +1,28 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=tiptop
+pkgver=2.3
+pkgrel=0
+pkgdesc="Tiptop: Hardware Performance Counters for the Masses"
+url="http://tiptop.gforge.inria.fr"
+arch="x86_64"
+license="gpl2"
+depends=""
+makedepends="byacc flex linux-headers ncurses-dev libxml2-dev"
+subpackages="${pkgname}-doc"
+source="
+	http://${pkgname}.gforge.inria.fr/releases/${pkgname}-${pkgver}.tar.gz
+"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr
+	make
+}
+
```

Comment:
It does have that, but it looks like any arg parsing happens after it tries to use the aforementioned sys call:
```
$ tiptop --help
syscall: Not supported
Could not perform syscall.
Don't know why...
```

Which is a hokey way to go about parsing arguments, but probably OK for x86, I think, it might depend on if you're running in a VM without hardware perf counters to access. We can try it?